raiden-dapp/src/components/SplashScreen.vue

```diff
@@ -77,6 +77,9 @@
             {{ $t('splash-screen.getting-started.link-name') }}
           </a>
         </i18n>
+        <div class="splash-screen__no-provider text-center font-weight-light">
+          <a @click="downloadLogs">{{ $t('splash-screen.download-logs') }}</a>
+        </div>
```

Comment:
Can we also offer a way to download the logs outside inside the dapp, not only from the splash screen? Some options:
* `ErrorDialog`
* Link in footer

Reply:
No, I'd prefer not, because then the logs would still be flowing. It's safer to do it when the SDK is stopped, i.e. splash screen, so no concurrent access while the file is being generated.

Reply:
AFAIK you can safely read stuff from IndexedDB while still being able to write new stuff.

Telling the user in case of errors to go to the connect screen to get logs seems very odd

Reply:
This feature isn't "to get logs in case of errors", if one wants to see it in real time, they can simply open DevTools, the place designed exactly for this kind of thing. It's an info gathering tool for bug reports, which is reasonable to ask do to with the SDK stopped, IMO.
Besides, I'm against adding it in multiple places, it'd just confuse the user when trying to tell them where to go fetch it, and clutch UI, and if it's to be in a single place, it needs to be on splash, so we are not locked out if some state bug prevents the user from connecting on the

Reply:
> This feature isn't "to get logs in case of errors"

I guess that is the exact use case for:

> It's an info gathering tool for bug reports

It's pretty common to give users option to collect the info when an error occurs

> it'd just confuse the user
> and clutch UI

I doubt that, in the end it's just a link

Reply:
I politely disagree with you. Please, if there's nothing technically wrong with the PR, approve it and open a feature **discussion** issue requesting and justifying buttons in multiple places to access this debugging tool. The PR fulfills the acceptance criteria, and I was not informed when first requested about this additional requirement.